path_config.go

```diff
@@ -104,9 +147,25 @@ func (b *backend) operationConfigCreateUpdate(ctx context.Context, req *logical.
 					}
 					certificates = append(certificates, certificate)
 				}
-				config.Certificates = certificates
+				config.IdentityCACertificates = certificates
 			case string:
```

Comment:
Aren't these always []string by virtue of the type being TypeStringSlice? You're making that assumption (correctly) above with `identityCACerts := data.Get("identity_ca_certificates").([]string)`

Reply:
Yes, I think I was overthinking it there. I did some more testing just to confirm that really is the behavior, and it all came back clean, so I just pushed an update simplifying the logic.